<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>601</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="todo">
             <property name="text">
              <string>display Todo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="display">
             <property name="text">
              <string>dispaly Interactions</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ajouter">
             <property name="text">
              <string>add Interaction</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre Contact : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="total">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>601</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="appliquer">
       <property name="text">
        <string>contacts entre deux dates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>les interactions entre deux dates</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="options">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAjouter_un_contact"/>
    <addaction name="actionRechercher_un_contact"/>
   </widget>
   <addaction name="options"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjouter_un_contact">
   <property name="text">
    <string>Ajouter un contact</string>
   </property>
  </action>
  <action name="actionRechercher_un_contact">
   <property name="text">
    <string>Rechercher un contact</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
